core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.js

```diff
@@ -0,0 +1,691 @@
+const navbarAboutTabBlogButton = 'a#navbar-about-menu-blog-button';
+const navbarGetInvolvedTab = 'a#navbar-get-involved-menu';
+const navbarGetInvolvedTabSchoolAndOrganizationsButton =
+  'a#navbar-get-involved-menu-school-and-organizations-button';
+const navbarGetInvolvedTabVolunteerButton =
+  'a#navbar-get-involved-menu-volunteer-button';
+const navbarGetInvolvedTabDonateButton =
+  'a#navbar-get-involved-menu-donate-button';
+const navbarGetInvolvedTabContactUsButton =
+  'a#navbar-get-involved-menu-contact-us-button';
+const navbarDonateButton = 'a#navbar-donate-button';
+
+const browseOurLessonsButton = '#about-page-browse-our-lessons-button';
+const accessAndroidAppButton = '#about-page-access-android-app-button';
+const visitClassroomButton = '#about-page-visit-classroom-button';
+const browseLibraryButton = '#about-page-browse-library-button';
+const createLessonsButton = '#about-page-create-lessons-button';
+const exploreLessonsButton = '#about-page-explore-lessons-button';
+
+const aboutFoundationClass = '.oppia-about-foundation-hero-content h1';
+const millionsOfContentId = '#about-foundation-page-millions-of-content';
+const weCannotContentId = '#about-foundation-page-we-cannot-content';
+const sourceUnescoButtion = 'a#about-foundation-page-source-unesco-button';
+const learnMoreAboutOppiaButton =
+  '#about-foundation-page-learn-more-about-oppia-button';
+const becomeAVolunteerButton =
+  '#about-foundation-page-become-a-volunteer-button';
+const sectionSixPart1 = '#about-foundation-page-section-six-part-1';
+const sectionSixPart2 = '#about-foundation-page-section-six-part-2';
+const sectionSixPart3 = '#about-foundation-page-section-six-part-3';
+
+const watchAVideoButton = 'a#thanks-for-donating-page-watch-a-video-button';
+const readOurBlogButton = 'a#thanks-for-donating-page-read-our-blog-button';
+const dismissButton = 'i#thanks-for-donating-page-dismiss-button';
+const thanksForDonatingClass = '.modal-open';
+const donatePage = '.modal-backdrop.fade';
+const thanksForDonatingPage = '.modal-backdrop.fade.show';
+
+module.exports = class LoggedInUsers extends baseUser {
+  /**
+   * Function for navigating to the home page.
+   */
+  async navigateToHome() {
+    await this.goto(homeUrl);
+  }
+
+  /**
+   * Function for navigating to the about page.
+   */
+  async navigateToAboutPage() {
+    await this.goto(aboutUrl);
+  }
+
+  /**
+   * Function for navigating to the about foundation page.
+   */
+  async navigateToAboutFoundationPage() {
+    await this.goto(aboutFoundationUrl);
+  }
+
+  /**
+   * Function for navigating to the Thanks for Donating page.
+   */
+  async navigateToThanksForDonatingPage() {
+    await this.page.goto(thanksForDonatingUrl, {
+      waitUntil: 'load',
+      timeout: 0
+    });
+  }
+
+  /**
+   * Function to click the About button in the About Menu on navbar
+   * and check if it opens the About page.
+   */
+  async clickAboutInAboutMenuOnNavbar() {
```

Comment:
Can you add the word "Button" in between "About" and "In" for the function name?